projects/go-lib/src/lib/components/go-off-canvas/go-off-canvas.component.scss

```diff
@@ -25,6 +25,22 @@
   z-index: z-index(off-canvas);
 }
 
+.go-off-canvas.large {
```

Comment:
In Goponents, we try to make use of BEM selector syntax for our CSS. That would transition this to a separate class declaration, something like `.go-off-canvas--large`

Also, ideally we do try to avoid nesting.. but I am okay with single nesting when it comes to modifier classes in BEM. Typically we've found that when declaring modifier classes it is best to put them towards the bottom of the file, simply because of the nature of CSS, but the main thing here is that they should be declared after the primary class it's modifying.

Reply:
I updated the selector.

I'm curious, why do we try to avoid nesting? Is that a BEM thing?

Reply:
Yeah, it's kind of a BEM thing. Additionally we've had issues with deep nesting in our application in the past so we try to avoid it in most cases.

Reply:
I moved the selector to the bottom of the file.

Would love to hear more about the nesting issues, but we can take that offline.